cfme/tests/control/test_bugs.py

```diff
@@ -481,3 +492,26 @@ def test_accordion_after_condition_creation(appliance, condition_class):
     assert view.conditions.tree.currently_selected == [
         "All Conditions", "{} Conditions".format(condition_class.TREE_NODE), condition.description
     ]
+
+
+@pytest.mark.meta(blockers=[BZ(1708434)], automates=[1708434])
+def test_edit_action_buttons(action_for_testing):
+    """
+    This tests the bug where the save/reset button are always enabled, even on initial load.
+
+    Bugzilla:
+        1708434
+
+    Polarion:
+        assignee: jdupuy
+        casecomponent: Control
+        caseimportance: medium
+        initialEstimate: 1/30h
+    """
+    view = navigate_to(action_for_testing, "Edit")
+
+    assert view.save_button.disabled
+    assert view.reset_button.disabled
+    # prep for deletion of the action during teardown
+    view.cancel_button.click()
+    navigate_to(action_for_testing, "Details")
```

Comment:
**Quesiton** Why not put this in `action_for_testing`?

Reply:
Good suggestion, I'll do that!